### DISCUSSION TITLE
Welcome to ResQFood Discussions!

### DISCUSSION THREAD

**Discussion by sourabhkumar47** [sourabhkumar47/ResQFood#21] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hemang-mishra** [comment]
I want to discuss the delivery of the food . Like, will it be from restaurants, charitable organizations, or volunteers. 

If we use the model of volunteers, there will be three stakeholders in our platform: a restaurant,  charitable organization, and volunteers. 

Please share if anything else is planned.

**Reply by sourabhkumar47** [reply]
You're thinking in wrong way, read readme carefully
